wlutil/launch.py

```diff
@@ -53,7 +53,6 @@ def getQemuCmd(config, initramfs=False):
     if 'img' in config and not initramfs:
         cmd = cmd + ['-device', 'virtio-blk-device,drive=hd0',
                      '-drive', 'file=' + config['img'] + ',format=raw,id=hd0']
-        cmd = cmd + ['-append', '"ro root=/dev/vda"']
```

Comment:
One side-effect here is that we don't need the 'root=' option anymore. The initramfs init script just looks in both places. The previous behavior was a bit of a hack anyway: we had to set CMDLINE_FALLBACK in the kernel to use the device-tree cmdline if present or the builtin if not, since Rocket doesn't provide one (amusingly, linux 5.2 has a broken CMDLINE_FALLBACK which cmdline.patch fixes)